consoles/localXvnc.pm

```diff
@@ -96,12 +89,14 @@ sub activate ($self) {
     start_xvnc($s, $display) unless $pid;
     close($s);
 
-    $self->connect_remote(
+    my $vnc = $self->connect_remote(
         {
             hostname => "localhost",
             port => $port,
             ikvm => 0
         });
+    # disable checking VNC stalls as this setup would not survive possible re-connects anyways
```

Comment:
As you put in a good explanation in the method "request_screen_update" that you remove with this change I suggest you move at least part of that explanation here to explain why.